blobfuse/blobfuse.cpp

```diff
@@ -47,7 +47,7 @@ const struct fuse_opt option_spec[] =
     OPTION("--use-attr-cache=%s", useAttrCache),
     OPTION("--use-adls=%s", use_adls),
     OPTION("--no-symlinks=%s", no_symlinks),    
-    OPTION("--no_symlinks=%s", no_symlinks),
+   // OPTION("--no_symlinks=%s", no_symlinks),
```

Comment:
This "_" option was add first and released and in later build we added new option with "-". In our Readme we have mentioned that both works, so I do not see any reason to remove this here.

Reply:
I think we should remove the extra _option from ReadMe.txt

Reply:
We can remove but just in case someone is using it then it will be a breakage for them,

Reply:
I see that we don't have it in the readme file

Reply:
By now, everyone should have fixed their links, I will add it to the revision notes

Reply:
Earlier it was there in Readme, I remember.  May be we cleaned it up in some commit, Fine then we can remove this.